### PR TITLE
Added details of partition by expression to the Chinese page

### PR DESCRIPTION
**Pull request by WujieRen**
英文官网不建议手动指定 partition by 字段对表进行分区，中文官网提都没提。

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
新增 partition by 表达式的详细解释到中文页。
（Added details of partition by expression to the Chinese page.）

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Review by vdimir — APPROVED** [review]
Thank you for your PR. Will merge right after signing CLA

**Comment by vdimir** [comment]
@WujieRen could you please sign CLA? https://github.com/ClickHouse/ClickHouse/pull/37510#issuecomment-1136707149

**Comment by alexey-milovidov** [comment]
Small corrections can be merged without CLA.
CLA Assistant service has poor usability for Chinese contributors.

**Pull request merged**
